.devcontainer/Wallaroo-Platform-DS/onCreateCommand.sh

```diff
@@ -1,27 +1,12 @@
-# wait for docker
-for i in $(seq 90); do
-    docker ps
-    status=$?
-    if [ "$status" -eq 0 ]; then
-        echo "docker worked after $(( i * 10 )) sec"
-        break
-    fi
-    printf '.'
-    sleep 10
-done
-docker ps
-status=$?
-[ "$status" -eq 0 ] || exit 1
-
-# # Add local registry https://kind.sigs.k8s.io/docs/user/local-registry/
-# # 1. Create registry container unless it already exists
-reg_name='kind-registry'
-reg_port='5001'
-if [ "$(docker inspect -f '{{.State.Running}}' "${reg_name}" 2>/dev/null || true)" != 'true' ]; then
-  docker run \
-    -d --restart=always -p "127.0.0.1:${reg_port}:5000" --name "${reg_name}" \
-    registry:2
-fi
+# workaround for https://github.com/kubernetes-sigs/kind/issues/3696
+# from https://github.com/dvordrova/kind-istio-tempo-otel/issues/1
+sudo tee /etc/docker/daemon.json > /dev/null <<'EOF'
+{
+  "ip6tables": false
+}
+EOF
+sudo pkill -f dockerd
+sudo dockerd --dns 168.63.129.16 &
```

Comment:
intentional static ip?

Reply:
Yes it's an azure IP https://learn.microsoft.com/en-us/azure/virtual-network/what-is-ip-address-168-63-129-16